app/views/hyrax/embargoes/_list_deactivated_embargoes.html.erb

```diff
@@ -0,0 +1,10 @@
+<% assets_with_deactivated_embargoes.each do |curation_concern| %>
+  <%= link_to curation_concern, edit_embargo_path(curation_concern) %> <%= visibility_badge(curation_concern.visibility) %>
+  <ul>
+    <% curation_concern.embargo_history.each do |entry| %>
+      <% unless entry.empty? %>
```

Comment:
That being said, I agree that `nil` doesn't respond to `empty?` and there is no actual need to have this logic in the block.  Could just do: 
```erb
<% curation_concern.embargo_history.select(&:present?).each do |entry| %>
```
Not sure I would hold up a PR this big/old over it though.